packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -86,3 +83,5 @@ export interface Metric<T> {
    */
   setCallback(fn: () => void): void;
 }
+
+export interface LabelSet {}
```

Comment:
What do you think about this being a `Map<string, string>` instead?

Reply:
One reason to used Object is because it's easy to get an array of the given object's values by using `Object.values()` . Another reason is we already know the structure is <string, string> in this scenario. What do you think?

Reply:
LabelSet cannot be an Object with no implementation because the `keys` must be unique. There is nothing in a your interface that guarantees this.

Reply:
Per the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics-user.md#label-set-calling-convention), SDK `Meter` should expose the method (`meter.Labels(...)`) to create the `LabelSet`, which will convert an unordered set of labels into a canonicalized set of labels, useful for pre-aggregation. WDYT?

Reply:
Oh good catch. @xiao-lix As @mayurkale22  has mentioned, the `Meter` should be able to construct `LabelSet` instances through `meter.Labels(...))`. In order to keep uniqueness, you can choose how to implement this (the logic in the meter function). You also have the freedom in the method signature (Map<string, string>, string[] string[], etc.). Map<string, string> seems to be good.